package/kernel/lantiq/ltq-ptm/src/ifxmips_ptm_fw_regs_vdsl.h

```diff
@@ -79,7 +79,18 @@ struct wan_tx_mib_table {
 
 #if defined(__BIG_ENDIAN)
 
-  struct fw_ver_id {
+  struct fw_ver_id_new {    //  @2000
+	/*  0 - 3h  */
+	unsigned int family         :4;
+    unsigned int package        :4;
+    unsigned int major          :8;
+    unsigned int middle         :8;
```

Comment:
Any reason for this strange indent?

Reply:
I tried to religiously follow the coding conventions (weird indents, whitespace etc.) found in the original driver code used as a reference here which is... not very well formatted indeed.

Reply:
@pesintta Please fix the indent, do `git commit -a --amend` and force-push onto the PR branch, so we can move forward with this. Thank you!

Reply:
@dangowrt thanks, I've now fixed the indent.